web/html/src/manager/image-profile-edit.js

```diff
@@ -105,17 +105,17 @@ class CreateImageProfile extends React.Component {
     });
   }
 
-  handleTokenChange(event) {
-    this.getChannels(event.target.value);
+  handleTokenChange(name, value) {
+    this.getChannels(value);
   }
 
-  handleImageTypeChange(event) {
-    const storeType = typeMap[event.target.value].storeType;
+  handleImageTypeChange(name, value) {
+    const storeType = typeMap[value].storeType;
     this.getImageStores(storeType);
   }
 
-  handleImageStoreChange(event) {
-    const storeLabel = event.target.value;
+  handleImageStoreChange(name, value) {
```

Comment:
Nitpick: you could also change the name of the function argument to `storeLabel` and avoid creating a constant.

Reply:
Fixed, and while at it I found another usage of `handleImageStoreChange()` that had the old signature